### ISSUE TITLE
XML error when visiting feed url

### ISSUE THREAD

**Issue by redgluten** [spatie/laravel-feed#3]
First let me thank you for the great work. :+1: 

I’ve got what seems to be an XML error when trying to visit my feed url (on Chrome at least, Safari and Firefox are only pointing to non-working feed): 

```
This page contains the following errors:

error on line 8 at column 45: Entity 'ecirc' not defined
Below is a rendering of the page up to the first error.
```

Any idea where to look?

This is my config file: 

```php
return [

    'feeds' => [
        [
            /*
             * Here you can specify which class and method will return
             * the items that should appear in the feed. For example:
             * '\App\Model@getAllFeedItems'
             */
            'items' => 'App\Article@getAllFeedItems',

            /*
             * The feed will be available on this url.
             */
            'url' => 'article-feed',

            'title' => 'Polenta ! - Articles',

            'description' => 'Tous les articles de Polenta !',
        ],
    ],

];
```

My Article model contains the necessary methods:

```php
    public function getFeedItemId()
    {
        return $this->id;
    }

    public function getFeedItemTitle() : string
    {
        return $this->title;
    }

    public function getFeedItemSummary() : string
    {
        return $this->title;
    }

    public function getFeedItemUpdated() : Carbon
    {
        return $this->updated_at;
    }

    public function getFeedItemLink() : string
    {
        return url('article.show', $this->id);
    }

    public function getAllFeedItems() : Collection
    {
        return Article::all();
    }
```

The Article title is just a simple UTF-8 string.

**Comment by redgluten** [comment]
This does look like an encoding bug, the entity “ecir” [represents the character](http://www.w3schools.com/charsets/ref_html_entities_e.asp) “ê” often used in french.

**Comment by freekmurze** [comment]
Could you post the url to your feed? We'll take a look at this problem in the coming days.

**Comment by redgluten** [comment]
It’s under local dev for the moment but I’ll try to deploy something in the next days.

**Comment by redgluten** [comment]
It took me some time but here it is: [http://polenta.glutendesign.com/article-feed](http://polenta.glutendesign.com/article-feed).

**Comment by freekmurze** [comment]
This should be fixed in v1.0.6. Of the package.

If you still encounter problems, reopen this issue.
